app/templates/suppliers/edit_supplier_trading_status.html

```diff
@@ -31,12 +33,27 @@
 <div class="single-question-page">
```

Comment:
We should be able to get rid of this wrapping div now that everything on this page is using `govuk-frontend` - this is an old class for spacing our frontend toolkit pages.

It may affect spacing a bit, but that's expected.